components/link/link.js

```diff
@@ -139,13 +156,31 @@ class Link extends FocusMixin(LitElement) {
 			'truncate': this.lines > 0
 		};
 		const styles = (this.lines > 0) ? { '-webkit-line-clamp': this.lines } : {};
+		if (this.newWindow && this.target === undefined) {
+			this.target = '_blank';
+		}
 		return html`<a
 				aria-label="${ifDefined(this.ariaLabel)}"
 				class="${classMap(linkClasses)}"
 				style="${styleMap(styles)}"
 				?download="${this.download}"
 				href="${ifDefined(this.href)}"
-				target="${ifDefined(this.target)}"><slot></slot></a>`;
+				target="${ifDefined(this.target)}"><slot></slot>${this.renderNewWindowIcon()}</a>`;
+	}
+
+	renderNewWindowIcon() {
+		const newWindowClasses = {
+			'd2l-new-window': true,
+			'd2l-icon-small': this.small,
+		};
+		return this.newWindow ? html`
+			<d2l-icon class="${classMap(newWindowClasses)}" aria-hidden="true" icon="tier1:new-window"></d2l-icon>
```

Comment:
`aria-hidden` isn't needed since `<d2l-icon>` doesn't actually render anything that screen readers will notice.
```suggestion
			<d2l-icon class="${classMap(newWindowClasses)}" icon="tier1:new-window"></d2l-icon>
```

Reply:
Narrator on windows says
"Link, Graphics, New Window Link, Opens in a new window." When the aria-hidden isn't included

With aira-hiddend it says
"Link, New Window Link" [Down Arrow Key] "Opens in a new window"

Personally I'd rather it say "graphic" and notify the user its a new window link all in one sentence than the aria-hidden version which breaks it up

Reply:
Interesting, I stand corrected. That's surprising though... I've never noticed this in other AT as we have `<d2l-icon>`s on their own a lot. I'd lean towards not putting `aria-hidden` on there just for consistency.